src/Microsoft.Health.Dicom.Core.UnitTests/Features/Store/DicomDatasetValidatorTests.cs

```diff
@@ -56,8 +56,7 @@ public async Task GivenDicomTagWithDifferentVR_WhenValidated_ThenShouldSkip()
             _queryTags.Add(new QueryTag(tag.BuildExtendedQueryTagStoreEntry()));
             IDicomElementMinimumValidator validator = Substitute.For<IDicomElementMinimumValidator>();
             _dicomDatasetValidator = new DicomDatasetValidator(featureConfiguration, validator, _queryTagService);
-            await _dicomDatasetValidator.ValidateAsync(_dicomDataset, requiredStudyInstanceUid: null);
-
+            await Assert.ThrowsAsync<DatasetValidationException>(() => _dicomDatasetValidator.ValidateAsync(_dicomDataset, requiredStudyInstanceUid: null));
```

Comment:
>DatasetValidationException [](start = 37, length = 26)

Can you validate the exception thrown?

Reply:
hi Sneha, do you mean verify exception message here?

Reply:
Yes!

---
In reply to: [619517233](https://github.com/microsoft/dicom-server/pull/736#discussion_r619517233) [](ancestors = 619517233)

Reply:
Updated to verify error message